sdk/eventgrid/eventgrid-namespaces/test.env

```diff
@@ -0,0 +1,22 @@
+# An Event Grid Topic configured to use the Event Grid Schema
+EVENT_GRID_EVENT_GRID_SCHEMA_ENDPOINT="https://<event grid topic name>.<event grid topic region>.eventgrid.azure.net/api/events"
```

Comment:
Are these env variables used anywhere, or should they be cleaned up?

Reply:
Done (in next commit)